lib/exceptions.dart

```diff
@@ -1,7 +1,7 @@
 part of 'torch_light.dart';
 
 class EnableTorchExistentUserException implements Exception {
-  String message;
+  String? message;
 
   EnableTorchExistentUserException(this.message);
```

Comment:
Given that now the `message` param is nullable, please change the incoming `message` parameter in the constructor to be optional.

This change can be applied to all of the exceptions of this file.

Also, it would be great to arrange the error message in case that the `message` is null.